FrenchQuizApp/Verbs/Views/AddVerbViewController.swift

```diff
@@ -0,0 +1,265 @@
+import UIKit
+import CoreData
+
+class AddVerbViewController: UIViewController, UITextFieldDelegate {
+    
+    var dataResultsController = ManagedData.verbResultsController
+    var managedObjectContext = ManagedData.persistentContainer.viewContext
+    var newVerbDictionary : [String : [String : String]] = [ : ]
+    let dispatchGroup = DispatchGroup()
+    
+    
+    @IBOutlet weak var englishTextField: UITextField!
+    @IBOutlet weak var frenchTextField: UITextField!
+    @IBOutlet weak var tableView: UITableView!
+    
+    @IBAction func searchWasPressed(_ sender: Any) {
+        if englishTextField.text == "" || frenchTextField.text == "" {
```

Comment:
in order to check for `nil` and empty strings `""`, try this:
if (englishTextField.text.isEmpty ?? true) || (frenchTextField.text.isEmpty ?? true)

Reply:
I didn't know that.  Thanks :)